.github/workflows/ubuntu.yml

```diff
@@ -182,10 +183,8 @@ jobs:
         # during compilation.
         run: |
           tar xfz archive.tar.gz
-          shopt -s dotglob
-          mv downward/* .
           if [[ ! -z "${CPLEX_URL}" || ! -z "${SOPLEX_URL}" ]]; then
-            mv lib/ ../
+            mv lib/ /home/runner
```

Comment:
Why don't we have to move downward (or however the repo is called) to the "right" place?

Reply:
Because the working dir is $GITHUB_WORKSPACE :-)